sway-fmt-v2/CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to the Sway Language Formatter
+
+Firstly, thank you for taking interest in advancing the Sway language formatter! This guide will walk you through installation and best practices for contributing to this project.
+
+## Installation
+
+In order to see `swayfmt` in action you will need both `swayfmt` and the [`forc-fmt`](../forc-plugins/forc-fmt/) plugin. As previously stated, you can install these via [`fuelup`](https://github.com/FuelLabs/fuelup) which will install the complete Fuel toolchain, or follow the below instructions to install from source.
+
+**note:** `cargo` is a pre-requisite to this build
+
+1. move to your preferred directory
+2. clone the Sway repo
+3. build from manifest and move the compiled result to your `.cargo/bin` folder
+
+```sh
+git clone https://github.com/FuelLabs/sway.git
+cargo build --manifest-path ~/sway/forc-plugins/forc-fmt/Cargo.toml
+mv ~/sway/target/debug/forc-fmt ~/.cargo/bin
+```
```

Comment:
When I read the points sequentially I wonder about point 3 and what it means to build some manifest and move some result to the specified directory. I think that should be elaborated by specifically mentioning the files i.e. what the manifest file is and what the result file is.

The subsequent section is a code block containing the steps.

I would prefer the following structure

1. Bullet Point 1

  ```bash
  $ command to move to dir
  ```

2. Bullet point 2

  ```bash
  $ clone command
  ```

etc.

I think this is much clearer to understand than multiple points and then 1 code block containing multiple steps e.g. bullet point 1 is omitted but bullet point 3 contains 2 commands; the last two lines of the block. As a beginner I might not be able to distinguish between the steps therefore separating into the aforemonetioned format makes it easier to see which command is performing which action